playground/src/context/theme-consumer.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import ThemeContext from './theme-context';
+
+interface ThemedButtonProps {}
```

Comment:
I would like to refactor all the `interface ThemedButtonProps` to just `type Props =`. I prefer that now as it is more concise and cleaner, also has a benefit to do things like this:
```ts
type Props = typeof dispatchProps & ReturnType<typeof mapStateToProps>;
```

Please could you do it for all the components related to this PR?